composer.json

```
@@ -0,0 +1,62 @@
+{
+    "name": "bcgov-theme/design-system-wordpress-child-theme-template",
+    "description": "Design System Wordpress Child Theme Template",
+    "version": "1.0.0",
+    "type": "wordpress-theme",
+    "license": "Apache-2.0",
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://apps.itsm.gov.bc.ca/satis"
+        }
+    ],
+    "require-dev": {
+        "bcgov/wordpress-utils": "@dev"
```

This should be set to ~2.5 which means it should be able to take any minor release >= 2.5 or < 3
```suggestion
        "bcgov/wordpress-utils": "~2.5"
```
If we don't do this, it causes issues in the future.